inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/config/model/instrumentation/InstrumentationSettings.java

```diff
@@ -31,4 +33,15 @@
      */
     private Map<String, Boolean> ignoredBootstrapPackages;
 
+    /**
+     * The configuration of the defined scopes. The map's key represents an unique id for the related instrumentation scope.
+     */
+    @Valid
+    private Map<String, InstrumentationScopeSettings> scopes;
```

Comment:
As discussed. This is separate because the scopes and DC (when we have data collectors) have a n:m relation to each other. Let's keep this for now and change if necessary.